docs/_documentations/mdt-eclipse-usingadifferenttemplate.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: Using a template repository of your own
-description: Using a template repository of your own
+title: Configuring template repositories
```

Comment:
Configuring template sources (other places too)

Reply:
It should be displaying as such in the latest commit. :)